src/cephadm/tests/test_cephadm.py

```diff
@@ -1079,6 +1079,19 @@ def test_get_version_prometheus(self, _call):
         version = cd.Monitoring.get_version(ctx, 'container_id', daemon_type)
         assert version == '0.16.1'
 
+    def test_prometheus_external_url(self):
+        ctx = cd.CephadmContext()
+        daemon_type = 'prometheus'
+        daemon_id = 'home'
+        fsid = 'aaf5a720-13fe-4a3b-82b9-2d99b7fd9704'
+        args = cd.get_daemon_args(ctx, fsid, daemon_type, daemon_id)
+        found = False
+        for arg in args:
+            if arg.startswith('--web.external-url=http://'):
+                found = True
+                break
+        assert found is True
```

Comment:
I think this block could be replaced by a shorter version, maybe something like:

`assert any([x.startswith('--web.external-url=http://') for x in args])`

Reply:
Right. I'm always going the long way because it's much more debug friendly.